performance_manager/lib/postgres_utils.py

```diff
@@ -93,13 +107,12 @@ def get_local_engine(
         db_name = os.environ.get("DB_NAME")
         db_password = os.environ.get("DB_PASSWORD", None)
         db_port = os.environ.get("DB_PORT")
-        db_region = os.environ.get("DB_REGION", None)
         db_user = os.environ.get("DB_USER")
         db_ssl_options = ""
 
         # when using docker, the db host env var will be "local_rds" but
-        # accessed via the "0.0.0.0" ip address
-        if db_host == "local_rds":
+        # accessed via the "0.0.0.0" ip address (mac specific)
+        if db_host == "local_rds" and "macos" in platform.platform().lower():
             db_host = "0.0.0.0"
```

Comment:
this one is just for me, the `0.0.0.0` is a mac specific docker quirk, so it won't happen now on linux